leasing/fixtures/batchrun_command.json

```diff
@@ -56,7 +56,47 @@
       "type": "django-manage",
       "name": "send_invoices_to_laske",
       "parameters": {},
-      "parameter_format_string": ""
+      "parameter_format_string": "1"
```

Comment:
What is the difference between parameters and parameter_format_string?

Reply:
I'm not sure, I haven't found any command that uses parameters so I decided to use parameter_format_string. There are a couple commands in database that use parameter_format_string: mvj_import (parameter_format_string: "area") and batchrun_log_rotate (parameter_format_string: "--dry-run")